tests/codegen.rs

```diff
@@ -375,13 +375,26 @@ impl From<&str> for TrustBits {
     }
 }
 
-const HEADER: &str = r#"//!
-//! This library is automatically generated from the Mozilla
-//! IncludedCACertificateReportPEMCSV report via ccadb.org. Don't edit it.
+const HEADER: &str = r#"//! A compiled-in copy of the root certificates trusted by Mozilla as of {:?}",
```

Comment:
I think this `{:?}` format placeholder is never run through a formatter, it's appearing in `lib.rs` as-is.

Reply:
Oops! An earlier version of this PR interpolated in the time, but I (mostly) reverted it when I realized I'd have to pull in `chrono` as a dev-dependency. I figured it could wait.